dd_launch

```diff
@@ -36,14 +36,13 @@ print "Liftoff!".
 wait until verticalSpeed > 10.
 
 local lock lookAt to heading(launchHeading, 90).
-local lock lookUp to heading(launchHeading, -45).
-lock steering to lookdirup(lookAt:vector, lookUp:vector).
+lock steering to lookdirup(lookAt:vector, SHIP:UP:vector).
```

Comment:
1. generally when contributing to another codebase, you should try to match their coding style - this should be `ship:up:vector`.
2. could you explain a little more what the problem you're seeing is? how does the navball not make sense?